aspnetcore/blazor/hybrid/static-files.md

```diff
@@ -0,0 +1,280 @@
+---
+title: ASP.NET Core Blazor Hybrid static files
+author: guardrex
+description: Learn how to configure and manage static files for Blazor Hybrid apps.
+monikerRange: '>= aspnetcore-6.0'
+ms.author: riande
+ms.custom: mvc
+ms.date: 05/18/2022
+no-loc: [".NET MAUI", "Mac Catalyst", "Blazor Hybrid", Home, Privacy, Kestrel, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+uid: blazor/hybrid/static-files
+---
+# ASP.NET Core Blazor Hybrid static files
+
+This article describes the configuration for serving static files in Blazor Hybrid apps.
+
+[!INCLUDE[](~/blazor/includes/blazor-hybrid-preview-notice.md)]
+
+In a Blazor Hybrid app, static files are *app resources*, accessed by Razor components using the following approaches:
+
+* [.NET MAUI](#net-maui): [:::no-loc text="Xamarin.Essentials":::](/xamarin/essentials/)
```

Comment:
Is a `Xamarin.Essentials` cross-link appropriate or should we link to something MAUI specific?

Reply:
Yes! ... I didn't get this reference correct. It's ".NET MAUI Essentials" if they kept the naming. The assembly is named that way (`Microsoft.Maui.Essentials.XXXXX`), and the namespace is `Microsoft.Maui.Storage`.

https://devblogs.microsoft.com/xamarin/xamarin-essentials-1-7-and-introducing-net-maui-essentials/#net-maui-essentials

I'll fix this up.